python/triton/compiler/compiler.py

```diff
@@ -72,6 +72,9 @@ def optimize_ttgir(mod, num_stages, arch):
     pm.add_tritongpu_remove_layout_conversions_pass()
     if isinstance(arch, int):
         pm.add_tritongpu_accelerate_matmul_pass(arch)
+    # TODO change interface of accelerate_matmul_pass
+    if torch.version.hip is not None and "USE_MFMA" in os.environ:
```

Comment:
@dfukalov FYI

Reply:
Yes, I prepared a simple refactoring of these checks. Will create new PR just after you merge this.